app/inbound/process-submitted-settlement.js

```diff
@@ -0,0 +1,23 @@
+const db = require('../data')
+const saveSettlement = require('./save-settlement')
+const getExistingSettlement = require('./get-existing-settlement')
+
+const processSubmittedSettlement = async (settlement) => {
+  const transaction = await db.sequelize.transaction()
+
+  try {
+    const existingSettlement = await getExistingSettlement(settlement, transaction)
```

Comment:
more of a suggestion than anything

for DB retrieval funcs I like to know what it is that we're using to find existing ones given by both the name and the parameters being passed to it

i.e. `getExistingSettlementByInvoiceNumberAndValue`; it's clear and most importantly (or somewhat importantly to me alone) it opens this up to becoming a pure function: `getExistingSettlementByInvoiceNumberAndValue(settlement.invoiceNumber, settlement.value)`

Reply:
naming things is hard but like this approach. Updated name and function